airlock_processor/shared_code/blob_operations.py

```diff
@@ -126,3 +125,12 @@ def get_blob_info_from_blob_url(blob_url: str) -> Tuple[str, str, str]:
 
 def get_blob_url(account_name: str, container_name: str, blob_name='') -> str:
     return f'{get_account_url(account_name)}{container_name}/{blob_name}'
+
+
+def get_storage_endpoint_suffix():
+    default_value = "core.windows.net"
+    try:
+        return os.environ["STORAGE_ENDPOINT_SUFFIX"]
+    except KeyError as e:
+        logging.warning(f"Missing environment variable: {e}. using default value: '{default_value}'")
```

Comment:
This is going to generate a warning log entry of every call to this method, maybe even on every azure function call. Are you sure about this?